site/content/release-notes.md

```diff
@@ -10,7 +10,12 @@ title = "Release Notes"
   of telemetry data and events from devices connected to LoRa network providers and/or
   LoRa gateways. Note that this adapter is *not* considered production ready yet.
   Any help in improving and enhancing the adapter is more than welcome.
-
+  
```

Comment:
please remove the empty line and format the paragraph analogously to the existing ones ...

Reply:
@sophokles73 I did it already and didn't push it. I just pushed those changes.